styleguide/pane.scss

```diff
@@ -96,20 +96,32 @@ $easeOutExpo: cubic-bezier(.190, 1.000, .220, 1.000);
   flex-flow: row wrap;
   overflow: auto;
   -webkit-overflow-scrolling: touch;
-  padding: 30px 20px 50px;
+  padding: 20px 20px 40px;
 }
 
 /* types of panes */
 
 /* publish */
 
 .kiln-toolbar-pane .messages {
+  padding-top: 20px;
+  position: relative;
+
   p {
     color: $black-50;
     margin: 0 0 10px;
   }
 }
 
+.kiln-toolbar-pane .publish-messages:before {
+  border-top: 1px solid $grey;
```

Comment:
Thoughts on having a modifier on `$grey`? Expect we may wind up with a few. Also, may be too early for this. We need to have a big think on what the next version of Kiln should look and feel like.